addons/website/static/src/js/menu/content.js

```diff
@@ -1114,7 +1114,7 @@ function _clonePage(pageId) {
             title: _t("Duplicate Page"),
             $content: $(qweb.render('website.duplicate_page_action_dialog')),
             confirm_callback: function () {
-                var new_page_name =  this.$('#page_name').val();
+                var new_page_name =  this.$content.find('#page_name').val();
```

Comment:
Nitpicking, you can ignore, but so you know:
You can fix linter error if you want on the line you touch
```suggestion
                var new_page_name = this.$content.find('#page_name').val();
```
And also update the `var` usage to `const`
```suggestion
                const new_page_name = this.$content.find('#page_name').val();
```
And finally probably also fix the naming linter error as this is only used once below:
```suggestion
                const newPageName = this.$content.find('#page_name').val();
```
At the end, the use of the variable could also be removed, so you'll just end up with
```js
                    args: [
                        pageId,
                        this.$content.find('#page_name').val(),
                    ],
```